shell/platform/tizen/flutter_tizen.cc

```diff
@@ -118,6 +118,13 @@ void FlutterDesktopMessengerSetCallback(FlutterDesktopMessengerRef messenger,
                                                             user_data);
 }
 
+#ifndef __X64_SHELL__
+void FlutterDesktopNotifyAppControl(FlutterDesktopEngineRef engine,
+                                    void* app_control) {
+  EngineFromHandle(engine)->app_control_channel->NotifyAppControl(app_control);
+}
+#endif
```

Comment:
```suggestion
void FlutterDesktopNotifyAppControl(FlutterDesktopEngineRef engine,
                                    void* app_control) {
#ifndef __X64_SHELL__
  EngineFromHandle(engine)->app_control_channel->NotifyAppControl(app_control);
#endif
}
```
`__X64_SHELL__` is only for internal use by the Tizen embedder, and it should not appear in the public embedder API. Please consider removing `#ifndef __X64_SHELL__` from `flutter_tizen.h` and making `FlutterDesktopNotifyAppControl` no-op for the desktop shell.